src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift

```diff
@@ -99,6 +112,7 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 	@OpenCombine.Published var isButtonHidden: Bool = true
 
 	@OpenCombine.Published var exposureNotificationError: ExposureNotificationError?
+	@OpenCombine.Published var appConfiguration: SAP_Internal_V2_ApplicationConfigurationIOS = SAP_Internal_V2_ApplicationConfigurationIOS()
```

Comment:
I think the ExposureDetectionViewModel should not be a publisher of appConfiguration. The only source should be the CachedAppConfiguration.